CMakeLists.txt

```diff
@@ -141,16 +141,6 @@ ExternalProject_Add(wrp-c
 add_library(libwrp-c STATIC SHARED IMPORTED)
 add_dependencies(libwrp-c wrp-c)
 
-# libparodus external dependency
-#-------------------------------------------------------------------------------
-ExternalProject_Add(libparodus
-    PREFIX ${CMAKE_CURRENT_BINARY_DIR}/_prefix/libparodus
-    GIT_REPOSITORY https://github.com/Comcast/libparodus.git
```

Comment:
This should not be deleted. libparodus is added as external dependency for libpd tests